pkg/controller/druid/handler.go

```diff
@@ -675,6 +675,15 @@ func updateDefaultPortInProbe(probe *v1.Probe, defaultPort int32) *v1.Probe {
 	return probe
 }
 
+func getTerminationGracePeriod(nodeSpec *v1alpha1.DruidNodeSpec) *int64 {
+
+	if nodeSpec.TerminationGracePeriodSeconds == 0 {
+		nodeSpec.TerminationGracePeriodSeconds = 30
```

Comment:
can we leave it be default value that k8s cluster would have instead of hardcoding 30 secs in the operator

Reply:
I had kept to absolute value, changed type to pointer.